modules/k6/k6.go

```diff
@@ -49,13 +54,27 @@ func SetEnvVar(variable string, value string) testcontainers.CustomizeRequestOpt
 	}
 }
 
-// WithCache uses the given volume as a cache for building the k6 binary.
-// If the volume does not exists, it is created.
-func WithCache(cache string) testcontainers.CustomizeRequestOption {
+// WithCache sets a volume as a cache for building the k6 binary
+// If a volume name is provided in the TC_K6_BUILD_CACHE, this volume is used.
```

Comment:
I think is more clear. Wdyt?
```suggestion
// If a volume name is provided in the TC_K6_BUILD_CACHE, this volume is used and it will persist across test sessions.
```